packages/amazon/src/serverGroup/configure/wizard/instanceType/InstanceTypeSelector.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useEffect, useState } from 'react';
+
+import { HelpField, IInstanceTypeCategory } from '@spinnaker/core';
+
+import { AdvancedModeSelector } from './advancedMode/AdvancedModeSelector';
+import { AWSProviderSettings } from '../../../../aws.settings';
+import { IAmazonInstanceTypeOverride, IAmazonServerGroupCommand } from '../../serverGroupConfiguration.service';
+import { SimpleModeSelector } from './simpleMode/SimpleModeSelector';
+
+export interface IInstanceTypeSelectorProps {
+  command: IAmazonServerGroupCommand;
+  setFieldValue: (field: keyof IAmazonServerGroupCommand, value: any, shouldValidate?: boolean) => void;
+  instanceTypeDetails: IInstanceTypeCategory[];
+}
+
+export function InstanceTypeSelector(props: IInstanceTypeSelectorProps) {
```

Comment:
nit: you can deconstruct the props so you don' t have to keep repeating `props.whatever` throughout the component.
```suggestion
export function InstanceTypeSelector({ command, setFieldValue }: IInstanceTypeSelectorProps) {
```